### PR TITLE
Explain CodeEditor parameters within docstring #10124

### LINKED ISSUE #10124: Explain CodeEditor's parameters within the doctring
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports

## Problem Description

There is not an explanation of the multiple parameters that a `CodeEditor` instance can receive

Following suggestion at https://github.com/spyder-ide/spyder/pull/9897#discussion_r320016309

### PR DESCRIPTION
**Pull request by Akashtyagi08**
## Description of Changes

* [x] Wrote at least one-line docstrings (for existing functions)

Added docstring explaining the parameters of `setup_editor()` function in `codeeditor.py`

### Issue(s) Resolved

Fixes #10124 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

### CODE REVIEW

**Comment by Akashtyagi08** [comment]
@dalthviz  Daniel, can you please guide me on this. Where am i doing wrong ? 

I am quite new to open source contribution.

**Comment by dalthviz** [comment]
Hi @Akashtyagi08 don't worry about the failing test, they are unrelated to your work here (@ccordoba12 and @goanpeca are working in a fix for that).

Also, seems like you did the changes using the `master` branch as base. If you want your changes to be released with the next version of Spyder `4.1.x` you will need to rebase this PR into the `4.x` branch. To do that, please follow the instructions in the contributing guide available [here](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md#changing-the-base-branch) and rebase your work into the `4.x` branch.

File: spyder/plugins/editor/widgets/codeeditor.py [hunk 1]
```diff
@@ -832,6 +832,81 @@ def setup_editor(self,
                      scroll_past_end=False,
                      debug_panel=True,
                      folding=True):
+        '''
```
Comment (dalthviz) [inline spyder/plugins/editor/widgets/codeeditor.py:835]: Please change the single quotes for double quotes: `'''` -> `"""`

File: spyder/plugins/editor/widgets/codeeditor.py [hunk 2]
```diff
@@ -832,6 +832,81 @@ def setup_editor(self,
...
+            Default 1500 ms.
+        show_class_func_dropdown: Enable/Disable a Matlab like widget to show
+            classes and functions available in the current file. Default False.
+        indent_guides: Enable/Disable highlighting of code indentation.
+            Default False.
+        scroll_past_end: Enable/Disable possibility to scroll file passed
+            its end. Default False.
+        debug_panel: Enable/Disable debug panel. Default True.
+        folding: Enable/Disable code folding. Default True.
+        '''
```
Comment (dalthviz) [inline spyder/plugins/editor/widgets/codeeditor.py:909]: Same here:  `'''` -> `"""`

**Comment by Akashtyagi08** [comment]
I have messed it up. Will be creating a new PR request .

**Pull request closed without merging**